src/main/java/com/NewsWorkflw/common/model/SampleContent.java

```diff
@@ -4,11 +4,25 @@
 
 public class SampleContent {
 
+	//Wouldn't we want to start putting in better names that will go with your design?

```

Comment:
Save this discussion for the pull request (where I am putting this comment; you can have inline conversations here).  It should not be in comments in the source code, because that's clutter, which is technical debt.